internal/ownerhandling/annotation_test.go

```diff
@@ -12,14 +12,40 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/utils/pointer"
-	"package-operator.run/package-operator/internal/testutil"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	"package-operator.run/package-operator/internal/testutil"
 )
 
+func TestOwnerStrategyAnnotation_RemoveOwner(t *testing.T) {
+	obj := &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "cm1",
+			Namespace: "test",
+			UID:       types.UID("1234"),
+			Annotations: map[string]string{
+				ownerStrategyAnnotation: `[{"uid":"123456", "kind":"ConfigMap", "name":"cm1"}]`,
+			},
+		},
+	}
+	owner := &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "cm1",
+			Namespace: obj.Namespace,
+			UID:       types.UID("123456"),
```

Comment:
```suggestion
```

Reply:
I think all the UID's in this file can now be deleted :)

Reply:
The current implementation is not needing them anymore, but they _should_ be present under normal conditions.
I would have just left them in there, because they don't hurt.

Reply:
:+1: